package.json

```diff
@@ -41,8 +42,9 @@
   "scripts": {
     "fix": "eslint . --fix",
```

Comment:
Oh interesting.  In other repos we do `eslint --fix '**/*.js'`.  Is this syntax equivalent?